content/kaiserschmarrn.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Kaiserschmarrn
+tags: ['sweets', 'sweet', 'german', 'austrian']
+date: 2023-04-30
+author: "Felix Steiner"
+---
+
+A very recoginzable and simple sweet dish from the mountainous regions of Austria, souther Germany and northern Italy.
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 10 min
+- 🍽️ Servings: 3
+
+## Ingredients
+
+- 200g plain flower
```

Comment:
what is "plain" flour? i guess you mean type-405 flour, which would be "pastry flour" [for americans](https://ellerepublic.de/en/german-vs-american-flour/)?

```suggestion
- 200g wheat flour (type 405 / pastry flour)
```